etc/restic/_global.env.sh

```diff
@@ -29,3 +29,8 @@ export RESTIC_BACKUP_EXTRA_ARGS=
 # Verbosity level from 0-3. 0 means no --verbose.
 # Override this value in a profile if needed.
 export RESTIC_VERBOSITY_LEVEL=0
+
+# (optional) Desktop notifications
```

Comment:
Maybe add `. See backup.sh` for details on how to set this up.`

Reply:
Done!